src/app/profile/update/update.component.spec.ts

```diff
@@ -358,26 +358,9 @@ describe('UpdateComponent', () => {
     });
   });
 
-  describe('#validateImageUrl', () => {
-    // Wrongfully fails
-    xit('should verify https://fabric8.io/ to be an invalid image url', () => {
-      let url: string = 'https://fabric8.io/';
-      component.imageUrl = url;
-      component.validateImageUrl();
-      expect(component.imageUrlInvalid).toBeTruthy();
-    });
-
-    it('should verify http://design.jboss.org/fabric8/logo/final/fabric8_logo_600px.png to be a valid image url', () => {
-      let url: string = 'http://design.jboss.org/fabric8/logo/final/fabric8_logo_600px.png';
-      component.imageUrl = url;
-      component.validateImageUrl();
-      expect(component.imageUrlInvalid).toBeFalsy();
-    });
-  });
-
   describe('#validateUrl', () => {
     it('should verify not-a-real-url to be an invalid url', () => {
-      let url: string = 'not-a-real-url';
+      let url: string = 'htt:/not-a-real-url.some-thing.';
```

Comment:
Could you add an array of strings containing various invalid urls and then run the inner 'test' over all of them?